components/google_drive/actions/create-file-from-template/create-file-from-template.mjs

```diff
@@ -0,0 +1,109 @@
+import googleDrive from "../../google_drive.app.mjs";
+import Mustaches from "google-docs-mustaches";
+
+const MODE_GOOGLE_DOC = 0;
+const MODE_PDF = 1;
+const MODE_GOOGLE_DOC_PDF = 2;
+
+export default {
+  key: "google_drive-create-file-from-template",
+  name: "Create New File From Template",
+  description: "Create a new google doc file from template",
```

Comment:
Actions should include a link to the relevant documentation. I would just link here to google-doc-mustaches:
```suggestion
  description: "Create a new google doc file from template. [See the docs here](https://www.npmjs.com/package/google-docs-mustaches)",
```